os/windows/env/env.ts

```diff
@@ -15,7 +15,13 @@ type IEnvironmentValueMatch = { groups: { name: string, type: string, data: stri
 
 const REG_KEY = 'HKEY_CURRENT_USER\\Environment'
 
-export async function setupWindowsEnvironmentPath (pnpmHomeDir: string, opts: { force: boolean } = { force: false }): Promise<string> {
+export interface SetupWindowsEnvironmentPathOpts {
+  envVarName?: string
+  prependedDir: string
```

Comment:
I would also rename this
as in this context, it's just dir

Reply:
The context is not too clear. Maybe if I would rename the function to something like `addDirToWinEnvPath`

Reply:
renamed the function and the variable.